bot/cogs/advent_of_code.py

```diff
@@ -74,6 +74,6 @@ async def leaderboard(self, ctx: Context) -> None:
                 await ctx.send(f"```  | Name {' '*(16-4)}| Stars | Score\n{msg}```")
 
 
-def setup(bot: Friendo) -> None:
+async def setup(bot: Friendo) -> None:
     """Sets up the AdventOfCode cog."""
-    bot.add_cog(AdventOfCode(bot))
+    await bot.add_cog(AdventOfCode(bot))
```

Comment:
Is loading cogs async now?

Reply:
Due to async magic, d.py v2 now utilizes async extension loading. Very helpful when a cog requires async tasks while loading.

Reply:
Very good.